.github/workflows/mysql-migtests.yml

```diff
@@ -60,12 +60,12 @@ jobs:
     - name: "TEST: mysql-constraints"
       run: migtests/scripts/run-test.sh mysql-tests/mysql-constraints
 
-    - name: "TEST: mysql-case-sensitive"
-      run: migtests/scripts/run-test.sh mysql-tests/mysql-case-sensitive
-
```

Comment:
Why?

Reply:
As the case-sensitive case doesn't come for MySQL as MySQL table names are itself case-sensitive and and it doesn't allow quotes. This is not required and I confirmed it from @shubham-yb  as well.